modules/store-devtools/src/provide-store-devtools.ts

```diff
@@ -19,9 +19,14 @@ import {
   StoreDevtoolsOptions,
 } from './config';
 import { ReducerManagerDispatcher, StateObservable } from '@ngrx/store';
-import { createStateObservable } from './instrument';
 import { StoreDevtools } from './devtools';
 
+export function createStateObservable(
```

Comment:
Arguably the export here is not required anymore.

```suggestion
function createStateObservable(
```

Reply:
Just like the other factory functions this can remain to be exported.